### ISSUE TITLE
Month view start on Monday

### ISSUE THREAD

**Issue by marcpursals** [twinssbc/Ionic-Calendar#130]
Hi, is there any chance to set Monday as first day of month view?

Thanks in advance

**Comment by twinssbc** [comment]
@marcpursals Yes, you can add "starting-day-month"=1 when declaring the calendar element in your HTML file.
